crates/bench_runner/src/main.rs

```diff
@@ -69,6 +69,8 @@ fn main() -> Result<()> {
             None,
             None,
             None,
+            Default::default(),
+            None,
```

Comment:
This function getting intense with all the Options.

Reply:
Yeah; I think there's a remote chance we can deprecate/consolidate `data_dir` and `service_account_key` into storage config so we'll be back to previous situation.